release-doc-in-process/all_vf.TTL

```diff
@@ -30,6 +30,23 @@ vf:AgentRelationshipRole  a  owl:Class ;
         rdfs:subClassOf   rdf:Property ;
         rdfs:comment   "A relationship role defining the kind of association one agent can have with another." .
 
+#vf:Recipe a                owl:Class ;
+#        rdfs:label         "vf:Recipe" ;
+#        rdfs:comment       "Recipes contain all the info required to create or improve or move a resource." .
+
+vf:ResourceSpecification a owl:Class ;
+        rdfs:label         "vf:ResourceSpecification" ;
+        rdfs:comment       "Specification of a kind of resource. Could define a material item, service, digital item, currency account, etc." .
+
+vf:RecipeProcess a          owl:Class ;
+        rdfs:label         "vf:RecipeProcess" ;
+        rdfs:comment       "Specifies the process part of a recipe for use in planning from recipe." .
+
+vf:RecipeFlow a            owl:Class ;
+        rdfs:label         "vf:RecipeFlow" ;
+        rdfs:comment       "The linkage between a resource type, process type, and action that create the structure of a recipe." .
```

Comment:
once again, we don't have *resource type* but *resource specification*, similar *recipe process* not *process type*

Reply:
I thought you wanted me to put it back, per the comment above.  Tell me the description you would like there please, I'm confused.

Reply:
Sorry, I meant to leave it the way you had it changed:
*"The linkage between a process recipe, an action that structures a recipe, and a resource specification."*

for now we will not say anything about some flow recipes possibly will not have reference to a resource specification.

Reply:
Thanks, done.